### DISCUSSION TITLE
Open links in new tab

### DISCUSSION THREAD

**Discussion by arastoonet** [withastro/starlight#1951] [Q&A] [answered]
Hi, How can I make the two main buttons on the homepage (**Get Started** and **View on GitHub**) open in a new tab (_blank)?

I've read a lot of documentation but haven't found anything. 
![btn](https://github.com/withastro/starlight/assets/39387375/cb2d48f8-7568-4a85-92fc-31dec49cdf7f)

**Comment by delucis** [comment]
The `actions` in the hero config can receive an optional `attrs` property to add arbitrary attributes as shown in the [`hero` frontmatter docs](https://starlight.astro.build/reference/frontmatter/#hero) for the `rel="me"` attribute.

If considering doing this, please do also carefully evaluate whether it is a good behaviour for users. A lot of user research recommends avoiding opening links in new windows, for example https://www.nngroup.com/articles/new-browser-windows-and-tabs/

**Reply by arastoonet** [reply]
I added:
`
attrs: 
        rel: me
`
but it didn't work. I also tried with `target: _blank` it didn't work too.

**Reply by HiDeoo** [reply, marked as answer]
> I added: `attrs: rel: me` but it didn't work. I also tried with `target: _blank` it didn't work too.

Maybe a syntax or formatting issue, here is an [example](https://stackblitz.com/edit/github-g29vtk?file=src%2Fcontent%2Fdocs%2Findex.mdx) where I added `target: _blank` to both hero actions to compare.

**Reply by arastoonet** [reply]
Finally :)) There was an additional (Enter) space in my code. I fix it. Thanks.
